client/src/document/index.scss

```
@@ -942,3 +942,41 @@ kbd {
     opacity: 0.4;
   }
 }
+
+html a.only-in-en-us:after {
+  content: "(en-US)";
+  font-size: smaller;
+  vertical-align: super;
+}
+
+html[lang="es"] a.only-in-en-us:after {
+  content: "(inglés)";
+}
+
+html[lang="fr"] a.only-in-en-us:after {
+  content: "(angl.)";
+}
+
+html[lang="ja"] a.only-in-en-us:after {
+  content: "(英語)";
+}
+
+html[lang="ko"] a.only-in-en-us:after {
+  content: "(영어)";
+}
+
+html[lang="ru"] a.only-in-en-us:after {
+  content: "(англ.)";
+}
+
+html[lang="pt-BR"] a.only-in-en-us:after {
+  content: "(inglês)";
+}
+
+html[lang="zh-CN"] a.only-in-en-us:after {
+  content: "（英语）";
+}
+
+html[lang="zh-TW"] a.only-in-en-us:after {
+  content: "（英語）";
+}
```

@yin1999 Just to double-check, is this what you're suggesting?

```suggestion
html[lang="zh-CN"] a.only-in-en-us:after {
  content: "（英语）";
  font-size: 80%;
}

html[lang="zh-TW"] a.only-in-en-us:after {
  content: "（英語）";
  font-size: 80%;
}
```

---

Hey @caugner. After further discussions with some other members on [discord](https://discord.com/channels/1009925603572600863/1233072948500369439), we decided aligning the text to the baseline :)

Not sure if there will be any other side-effect.

```suggestion
html[lang="zh-CN"] a.only-in-en-us:after {
  content: "（英语）";
  vertical-align: baseline;
}

html[lang="zh-TW"] a.only-in-en-us:after {
  content: "（英語）";
  vertical-align: baseline;
}
```